cot/src/form.rs

```diff
@@ -507,43 +615,153 @@ pub trait AsFormField {
 mod tests {
     use bytes::Bytes;
 
+    use super::*;
     use crate::Body;
-    use crate::form::form_data;
-    use crate::headers::FORM_CONTENT_TYPE;
+    use crate::headers::{MULTIPART_FORM_CONTENT_TYPE, URLENCODED_FORM_CONTENT_TYPE};
 
     #[cot::test]
-    async fn form_data_extract_get_empty() {
+    async fn urlencoded_form_data_extract_get_empty() {
```

Comment:
Please add as well tests for HEAD and something else to test the error case

Reply:
Done!